dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -752,10 +796,10 @@ paths:
           type: integer
           format: int64
       responses:
-        default:
-          content:
-            application/json: {}
```

Comment:
this response type seems to have lost its json nature, might be worth taking a look

Reply:
I don't think it lost its json nature. The default response for this endpoint is a 204 No content, so `application/json: {}` is just indicating that there's no content. I could add into the annotation that it explicitly returns no content so `application/json: {}` shows up, but it's a little redundant